kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -214,14 +214,26 @@ public class ResponsiveConfig extends AbstractConfig {
       + "will not be enabled. Setting this to a positive integer will enable async processing, but only if "
       + "there is at least one AsyncProcessor in the topology. See javadocs for AsyncProcessorSupplier for details.";
 
-  public static final String ASYNC_MAX_EVENTS_PER_KEY_CONFIG = "responsive.async.max.events.per.key";
-  private static final int ASYNC_MAX_EVENTS_PER_KEY_DEFAULT = 5;
-  private static final String ASYNC_MAX_EVENTS_PER_KEY_DOC = "The maximum number of pending events at a time for "
-      + "a given key. This puts a limit on how many events for each key can be either currently processing or awaiting "
-      + "processing by the async thread pool. Since all events with the same key must be processed in offset order, this"
+  public static final String ASYNC_MAX_EVENTS_QUEUED_PER_ASYNC_THREAD_CONFIG = "responsive.async.max.events.queued.per.async.thread";
```

Comment:
this is the new config -- I thought it made sense to just make it a multiplicative factor instead of the absolute event cap, but I worry this might go over people's heads? Do the docs/config name make this clear enough? Happy to accept naming suggestions

Reply:
This makes sense to me - it describes exactly what it is.